tools/convert/templates/configV2.tmpl

```diff
@@ -749,11 +749,12 @@ Collection:
     ## MaxAlloc is the maximum number of bytes that should be allocated by
     ## the collector.
     ##
-    ## If set, it must be an integer >= 0. 64-bit values are supported. See
-    ## MaxMemory for more details.
+    ## If set, this must be a memory size. 64-bit values are supported. Sizes
+    ## with standard unit suffixes like "MB" and "GiB" are also supported.
+    ## See MaxMemory for more details.
```

Comment:
I think it would be good to either list out to link to the complete list of valid suffixes

Reply:
Done and regenerated.